src/Components/Facility/SetInventoryForm.tsx

```diff
@@ -175,6 +174,7 @@ export const SetInventoryForm = (props: any) => {
                     type="string"
                     value={currentUnit}
                     errors=""
+                    disabled={true}
```

Comment:
If we are setting this as disabled always, why should this be an input field, can we design it better?

Reply:
It conveys to me that this field was manually set (input) by a user and is no longer editable. More like, "This field is not available for editing". Do you recommend converting it to a plain text?

Reply:
@Ashesh3 can we do something like a badge maybe
![image](https://user-images.githubusercontent.com/29787772/203707624-c7bc9b83-d3a0-4e79-8434-9b947cc83cff.png)

@nihal467 what's your take

Reply:
@khavinshankar what it up in the EOD